src/core/current_task.cc

```diff
@@ -0,0 +1,11 @@
+#include <seastar/core/reactor.hh>
+#include <seastar/core/current_task.hh>
+
+namespace seastar {
```

Comment:
This should use the `seastar::engine()` and `seastar::reactor::current_task()` functions.